filters/resources.txt

```diff
@@ -2046,3 +2046,67 @@ set-constant.js application/javascript
 		}
 	});
 })();
+
+
+# This removes BODY & HEAD elements if run inside framed content.
+remove-frame-content.js application/javascript
+(function() {
+	var log = console.log.bind(console);
+	var removedHead = {
+		async: false,
+		result: false
+	};
+	var removedBody = removedHead;
```

Comment:
This will not copy the object, both variable will point to the same object in memory.

Reply:
Thanks!